astropy/visualization/wcsaxes/tests/test_images.py

```diff
@@ -109,6 +109,9 @@ def test_contour_overlay(self):
         ax.set_xlim(0., 720.)
         ax.set_ylim(0., 720.)
 
+        # For backward-compatibility with previous reference images
```

Comment:
Maybe explain in this comment what change required this?

Reply:
Done